lfs/config_test.go

```diff
@@ -102,11 +102,12 @@ func TestSSHEndpointOverridden(t *testing.T) {
 	assert.Equal(t, "lfs", endpoint.Url)
 	assert.Equal(t, "", endpoint.SshUserAndHost)
 	assert.Equal(t, "", endpoint.SshPath)
+	assert.Equal(t, "", endpoint.SshPort)
 }
 
 func TestSSHEndpointAddsLfsSuffix(t *testing.T) {
 	config := &Configuration{
-		gitConfig: map[string]string{"remote.origin.url": "git@example.com:foo/bar"},
+		gitConfig: map[string]string{"remote.origin.url": "ssh://git@example.com/foo/bar"},
```

Comment:
I actually did do that - it's just a renaming of the test to differentiate between ssh:// and bare URLs. This test is now called "TestBareSSHEndpointAddsLfsSuffix" instead.